packages/autocomplete-client/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "@meilisearch/autocomplete-client",
+  "version": "0.1.0",
+  "private": false,
+  "description": "The search client to use Meilisearch with autocomplete.js.",
+  "scripts": {
+    "clear_jest": "jest --clearCache",
+    "cleanup": "shx rm -rf dist/",
+    "test:watch": "yarn test --watchAll",
+    "test": "jest --runInBand --selectProjects dom --selectProjects node",
+    "build": "yarn cleanup && rollup -c rollup.config.js && rollup --environment NODE_ENV:production -c rollup.config.js",
+    "dev": "rollup -c rollup.config.js --watch",
+    "lint": "eslint --ext .js,.ts,.tsx .",
+    "lint:fix": "eslint --ext .js,.ts,.tsx . --fix",
+    "test:types": "yarn tsc",
+    "version": "node scripts/update_version.js"
+  },
+  "main": "./dist/autocomplete-client.umd.js",
+  "module": "./dist/autocomplete-client.esm.js",
+  "browser": "./dist/autocomplete-client.umd.js",
+  "cjs": "./dist/autocomplete-client.cjs.js",
+  "source": "src/index.ts",
+  "typings": "./dist/types/index.d.ts",
+  "types": "./dist/types/index.d.ts",
+  "sideEffects": false,
+  "bugs": {
+    "url": "https://github.com/meilisearch/instant-meilisearch/issues"
```

Comment:
Is this URL still correct? 🤔

Reply:
I will be changed in another PR when I change the repo's name! Currently since the repository is still named `instant-meilisearch`, the link is still the correct one :)